spec/index.bs

```diff
@@ -1205,8 +1224,40 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
     <tr class=row-continuation><td colspan=4>
         The maximum allowed {{GPUVertexBufferLayout/arrayStride}}
         when creating a {{GPURenderPipeline}}.
+
+    <tr><td><dfn>maxInterStageShaderLocations</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>64
```

Comment:
I thought this would be 60 for now until we figure out the position thing for sure?